### PR TITLE
[tftrt update]

### PR DESCRIPTION
**Pull request by jjsjann123**
code cleaning, removed some boilerplate code

### CODE REVIEW

**Comment by jjsjann123** [comment]
Ping @aaroey

File: tensorflow/contrib/tensorrt/convert/convert_nodes.cc [hunk 1]
```diff
@@ -2214,64 +2213,62 @@ tensorflow::Status ConvertCalibrationNodeToEngineNode(
...
+      order->push_front(node);
     }
   }
-  // topological order is needed to build TRT network
-  static int static_id = 0;
+  return tensorflow::Status::OK();
+}
+
+tensorflow::Status SetInputList(
+    tensorrt::convert::SubGraphParams& s,
```
Comment (aaroey) [inline tensorflow/contrib/tensorrt/convert/convert_nodes.cc]: Use const reference instead?

**Review by aaroey — COMMENTED** [review]
Thanks for the refactoring!
Would you help to test the change against test:tf_trt_integration_test and test/test_tftrt.py?

**Comment by jjsjann123** [comment]
I was testing with my model ci_test.
Will run against tf_trt_integration_test and test_tftrt.py

**Comment by jjsjann123** [comment]
test run seems to be doing fine.
tf_trt_integration test is reporting `Ran 4 tests OK (skipped=1)`

**Review by aaroey — APPROVED** [review]

**Comment by rmlarsen** [comment]
@aaroey Thanks for the review!

**Pull request merged**